diff --git a/budgie-wpreviews/src/previews_controls.vala b/budgie-wpreviews/src/previews_controls.vala
index 9f8daf5..e8a15a0 100644
--- a/budgie-wpreviews/src/previews_controls.vala
+++ b/budgie-wpreviews/src/previews_controls.vala
@@ -120,7 +120,7 @@ namespace PreviewsControls {
         }
 
         private bool procruns (string processname) {
-            string cmd = @"/usr/bin/pgrep -f $processname";
+            string cmd = @"/bin/pgrep -f $processname";
             string output;
             try {
                 GLib.Process.spawn_command_line_sync(cmd, out output);
@@ -168,4 +168,4 @@ namespace PreviewsControls {
         controls.show_all();
         Gtk.main();
     }
-}
\ No newline at end of file
+}
-- 
2.26.0

