From ffce8f1b3f515f79bb6b1c37ee40bc32dca08422 Mon Sep 17 00:00:00 2001
From: Justin Zobel <justin.zobel@gmail.com>
Date: Wed, 3 Jun 2020 16:50:49 +0930
Subject: [PATCH] Fix monitor config launch issue

---
 startlxqt.in | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/startlxqt.in b/startlxqt.in
index bf50323..700f310 100755
--- a/startlxqt.in
+++ b/startlxqt.in
@@ -90,5 +90,13 @@ export XDG_MENU_PREFIX="lxqt-"
 
 export XDG_CURRENT_DESKTOP="LXQt"
 
+# Fix launch of lxqt monitor config: https://github.com/lxqt/lxqt/wiki/ConfigLaunch
+if [ -d /etc/X11/xinit/xinitrc.d ] ; then
+ for f in /etc/X11/xinit/xinitrc.d/?*.sh ; do
+  [ -x "$f" ] && . "$f"
+ done
+ unset f
+fi
+
 # Start the LXQt session
 exec lxqt-session
-- 
2.26.2

