From 163b78a6ecb6e90a7f5c9bd662f5d5e63b748fd6 Mon Sep 17 00:00:00 2001
From: Algent Albrahimi <algent@protonmail.com>
Date: Thu, 1 Oct 2020 20:15:26 +0200
Subject: [PATCH] hardcode pgrep path for hotcorners

---
 budgie-hotcorners/src/HotCornersApplet.vala | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/budgie-hotcorners/src/HotCornersApplet.vala b/budgie-hotcorners/src/HotCornersApplet.vala
index aebf538..f80cfeb 100644
--- a/budgie-hotcorners/src/HotCornersApplet.vala
+++ b/budgie-hotcorners/src/HotCornersApplet.vala
@@ -28,7 +28,7 @@ namespace HCSupport {
     */
 
     private bool locked () {
-        string cmd = Config.PACKAGE_BINDIR + "/pgrep -f gnome-screensaver-dialog";
+        string cmd = "/bin/pgrep -f gnome-screensaver-dialog";
         string output;
         try {
             GLib.Process.spawn_command_line_sync(cmd, out output);
@@ -429,7 +429,7 @@ namespace HotCornersApplet {
     }
 
     private bool procruns (string processname) {
-        string cmd = Config.PACKAGE_BINDIR + "/pgrep -f $processname";
+        string cmd = @"/bin/pgrep -f $processname";
         string output;
         try {
             GLib.Process.spawn_command_line_sync(cmd, out output);
-- 
2.28.0

