From 2b0bc88c64d78ecfabc47db4974f59a08e466987 Mon Sep 17 00:00:00 2001
From: William Floyd <william.png2000@gmail.com>
Date: Sat, 6 Oct 2018 11:54:17 -0500
Subject: [PATCH] autogen: Remove exit for autogen check.

---
 autogen.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 10b4dd34..d7be83ae 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -31,7 +31,6 @@ if test -z "$AUTOMAKE"; then
 	echo "You must have one of automake $AUTOMAKE_VERSIONS to compile $PROJECT."
 	echo "Install the appropriate package for your distribution,"
 	echo "or get the source tarball at http://ftp.gnu.org/gnu/automake/"
-	exit 1
 fi
 
 (gtkdocize --version) < /dev/null > /dev/null 2>&1 || {
-- 
2.18.0

