From 308d998fa6d9f3fdd036e9d9c99e0e38c2faa49e Mon Sep 17 00:00:00 2001
From: William Floyd <william.png2000@gmail.com>
Date: Sat, 19 May 2018 15:26:54 -0400
Subject: [PATCH] CMakeLists: Remove specific Glib version.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9d57aca..bfb01c4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,7 +10,7 @@ include(ValaVersion)
 find_package(Vala)
 ensure_vala_version("0.22.0" MINIMUM)
 find_package(PkgConfig)
-find_package(GLib "2.48" REQUIRED)
+find_package(GLib REQUIRED)
 find_package(JsonGlib)
 find_package(Libsecret)
 find_package(FontConfig)
-- 
2.17.0

