From 1c812739b25c3282e1908ea2060d8b390c164965 Mon Sep 17 00:00:00 2001
From: William Floyd <william.png2000@gmail.com>
Date: Fri, 1 Mar 2019 20:27:45 -0600
Subject: [PATCH 1/1] Makefile: Make directory

---
 Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile b/Makefile
index 46425b8..6dadda7 100644
--- a/Makefile
+++ b/Makefile
@@ -18,9 +18,13 @@ generator:
 	cd src/gir-generator && ${GOBUILD}  -o $(CURRENT_DIR)/${GENERATOR}
 
 copyfile:
+	mkdir -p $(OUT_GIR_DIR)gobject-2.0
 	cp -r  lib.in/gobject-2.0/*   $(OUT_GIR_DIR)gobject-2.0
+	mkdir -p $(OUT_GIR_DIR)gio-2.0
 	cp -r  lib.in/gio-2.0/*       $(OUT_GIR_DIR)gio-2.0
+	mkdir -p $(OUT_GIR_DIR)glib-2.0
 	cp -r  lib.in/glib-2.0/*      $(OUT_GIR_DIR)glib-2.0
+	mkdir -p $(OUT_GIR_DIR)gudev-1.0
 	cp -r  lib.in/gudev-1.0/*     $(OUT_GIR_DIR)gudev-1.0
 
 glib-2.0: lib.in/glib-2.0/glib.go.in lib.in/glib-2.0/config.json generator
-- 
2.20.1

