--- a/panel-plugin/notification-plugin.c
+++ b/panel-plugin/notification-plugin.c
@@ -27,7 +27,7 @@
 #include <gtk/gtk.h>
 #include <xfconf/xfconf.h>
 #include <libxfce4util/libxfce4util.h>
-#include <libxfce4panel/xfce-panel-plugin.h>
+#include <libxfce4panel/libxfce4panel.h>
 
 #include "notification-plugin.h"
 #include "notification-plugin-log.h"
@@ -201,7 +201,7 @@
   gboolean               state;
 
   /* Allocate memory for the plugin structure */
-  notification_plugin = panel_slice_new0 (NotificationPlugin);
+  notification_plugin = g_slice_new0 (NotificationPlugin);
   notification_plugin->plugin = panel_plugin;
 
   /* Initialize xfconf */
@@ -271,7 +271,7 @@
     g_source_remove (notification_plugin->menu_size_allocate_next_handler);
 
   /* free the plugin structure */
-  panel_slice_free (NotificationPlugin, notification_plugin);
+  g_slice_free (NotificationPlugin, notification_plugin);
 }
 
 
--- a/panel-plugin/notification-plugin-dialogs.c
+++ b/panel-plugin/notification-plugin-dialogs.c
@@ -26,7 +26,7 @@
 
 #include <xfconf/xfconf.h>
 #include <libxfce4ui/libxfce4ui.h>
-#include <libxfce4panel/xfce-panel-plugin.h>
+#include <libxfce4panel/libxfce4panel.h>
 
 #include "notification-plugin.h"
 #include "notification-plugin-log.h"
--- a/panel-plugin/notification-plugin-log.c
+++ b/panel-plugin/notification-plugin-log.c
@@ -32,7 +32,7 @@
 #include <gtk/gtk.h>
 #include <xfconf/xfconf.h>
 #include <libxfce4util/libxfce4util.h>
-#include <libxfce4panel/xfce-panel-plugin.h>
+#include <libxfce4panel/libxfce4panel.h>
 
 #include <common/xfce-notify-log.h>
 
