commit d87a90089b83180779ad6668e1e2bb49b0081996
Author: William Floyd <william.png2000@gmail.com>
Date:   Wed Mar 21 13:55:52 2018 -0400

    platformplugin/linux: Force QT location.

diff --git a/platformplugin/linux.pri b/platformplugin/linux.pri
index bfe8d41..f95cce0 100644
--- a/platformplugin/linux.pri
+++ b/platformplugin/linux.pri
@@ -69,5 +69,5 @@ exists($$PWD/libqt5xcbqpa-dev/$$QT_VERSION) {
 } else:exists($$[QT_INSTALL_HEADERS]/QtXcb/$$[QT_VERSION]) {
     INCLUDEPATH += $$[QT_INSTALL_HEADERS]/QtXcb/$$[QT_VERSION]/QtXcb/private
 } else {
-    error(Not support Qt Version: $$QT_VERSION)
+    INCLUDEPATH += /usr/include/qtxcb-private
 }
