commit a1ac72b2566c85d2b6316a20739b48577077334f
Author: William Floyd <william.png2000@gmail.com>
Date:   Tue Mar 20 20:33:45 2018 -0400

    Makefile: Use LIBDIR.

diff --git a/Makefile b/Makefile
index 9a6a92e..f672e50 100644
--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@ install:
 	mkdir -p ${DESTDIR}/usr/share/backgrounds/deepin
 	install -Dm644 files/i18n_dependent.json  ${DESTDIR}/usr/share/i18n/i18n_dependent.json
 	install -Dm644 files/language_info.json  ${DESTDIR}/usr/share/i18n/language_info.json
-	install -Dm644 files/desktop-version ${DESTDIR}/usr/lib/deepin/desktop-version
+	install -Dm644 files/desktop-version ${DESTDIR}${LIBDIR}/deepin/desktop-version
 	install -Dm644 files/lsb-release     ${DESTDIR}/etc/lsb-release
 	install -Dm644 files/appstore.json     ${DESTDIR}/etc/appstore.json
 	install -Dm644 files/deepin-logo.png ${DESTDIR}/usr/share/plymouth/deepin-logo.png
