--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -43,7 +43,7 @@
 #~
 
 add_custom_target(json_i18n ALL
-    /usr/lib/dtk2/dtk-settings resources/data/settings.json -o common/settings_translation.cpp
+    /usr/lib64/libdtk-2.1.1/DCore/bin/dtk-settings resources/data/settings.json -o common/settings_translation.cpp
     COMMAND sed -i "s,#include .*,#include <DSettings>," common/settings_translation.cpp
     VERBATIM
     SOURCES resources/data/settings.json
