--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -51,7 +51,7 @@
 set (DTK_SETTINGS_TOOLS_EXECUTABLE ${DTKCORE_TOOL_DIR}/dtk-settings)
 if (EXISTS ${DTK_SETTINGS_TOOLS_EXECUTABLE})
 add_custom_target(json_i18n ALL
-        ${DTKCORE_TOOL_DIR}/dtk-settings resources/data/settings.json -o common/settings_translation.cpp
+        /usr/lib64/libdtk-2.1.1/DCore/bin/dtk-settings resources/data/settings.json -o common/settings_translation.cpp
     COMMAND sed -i "s,#include .*,#include <DSettings>," common/settings_translation.cpp
     VERBATIM
     SOURCES resources/data/settings.json
